modules/javafx.base/src/test/java/test/javafx/collections/VetoableObservableListTest.java

```diff
@@ -89,107 +89,108 @@ protected void onProposedChange(List<String> added, int... removed) {
     }
 
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
```

Comment:
You need to replace the annotation with an `assertThrows` call in the method. Otherwise, if the test ever gets re-enabled it will not do what we expect.

This same comment applies to several other methods in this class.